front/css.php

```diff
@@ -59,6 +60,17 @@
 // Ensure warnings will not break CSS output.
 ErrorHandler::getInstance()->disableOutput();
 
+// If a custom theme is requested, we need to get the real path of the theme
+if (isset($_GET['file']) && isset($_GET['is_custom_theme']) && $_GET['is_custom_theme']) {
+    $theme = ThemeManager::getInstance()->getTheme($_GET['file']);
+
+    if (!$theme) {
```

Comment:
Maybe a warning could be added. Without it, the administrator will not be able to understand why its custom theme is not rendered.

```suggestion
    if (!$theme) {
        trigger_error(sprintf('Unable to find theme `%s`.', $_GET['file']), E_USER_WARNING);
```